### ISSUE TITLE
Pretty high barrier here

### ISSUE THREAD

**Issue by kb1ooo** [AnilOsmanTur/video_anomaly_diffusion#7]
I know that this code is provided open source and there should be no real expectation of support, but there really isn't enough to go on here to understand what the protocol, sequence of steps, etc is for training models with this repo.  And there doesn't seem to be anyone responding to issues.  Is this repo meant to just archive the code from the paper?

**Comment by iyc2** [comment]
Hello, I have been running this code recently. Have you resolved these issues?

**Comment by zuiwomeirenxi** [comment]
> Hello, I have been running this code recently. Have you resolved these issues?

Have you successfully run the code?

**Comment by iyc2** [comment]
> code?

I am debugging the code in another paper published by the author in ICIAP2023. If you encounter any problems in this paper, or you can run another paper, I think there are significant similarities between the two.

**Comment by kb1ooo** [comment]
> Hello, I have been running this code recently. Have you resolved these issues?

No, I haven't.  Any info you could provide would be very helpful.

**Comment by zuiwomeirenxi** [comment]
> > code?
> 
> I am debugging the code in another paper published by the author in ICIAP2023. If you encounter any problems in this paper, or you can run another paper, I think there are significant similarities between the two.

Could you please provide the features of UCF-Crime's 3D-ResNeXt-18 if you have them?"I only have an RTX 2060 Super, and extracting features from UCF-Crime is too slow."
